Modules/IO/NIFTI/src/itkNiftiImageIO.cxx

```diff
@@ -1753,21 +1753,26 @@ Normalize(std::vector<double> & x)
 static bool
 IsAffine(const mat44 & nifti_mat)
 {
-  const vnl_matrix_fixed<float, 4, 4> mat(&(nifti_mat.m[0][0]));
+  vnl_matrix_fixed<double, 4, 4> mat;
+
+  for (int i = 0; i < 4; i++)
+    for (int j = 0; j < 4; j++)
+      mat[i][j] = (double)nifti_mat.m[i][j];
```

Comment:
Please either just remove the C-style `(double)` cast here (because the conversion from `float` to `double` can take place safely implicitly), or replace it with the C++11/14 `T{expression}` syntax, using curly braces:

    mat[i][j] = double{ nifti_mat.m[i][j] }; 

Following the C++ Core Guidelines, which discourages C-style `(T)expression` casts, at:
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#prosafety-type-safety-profile